manifest: Update the Matter SDK version to 1.1.0.1 TAG (#11084)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  Generated onboarding codes should be put on the device's package or on the device itself.
  For details, see the Generating onboarding codes section on the :doc:`matter:nrfconnect_factory_data_configuration` page in the Matter documentation.
* Introduced ``SLEEPY_ACTIVE_THRESHOLD`` parameter that makes the Matter sleepy device stay awake for a specified amount of time after network activity.

> * Added device finish and device color attributes to the Basic Information cluster and added the proper entries in factory data set.

Reviewer: ```suggestion
* Updated the Basic Information cluster with device finish and device color attributes and added the related entries in factory data set.
```
